tests/system/test_gbq.py

```diff
@@ -426,33 +426,6 @@ def test_query_inside_configuration(self, project_id):
         )
         tm.assert_frame_equal(df, DataFrame({"valid_string": ["PI"]}))
 
-    def test_configuration_without_query(self, project_id):
```

Comment:
I'm not 100% sure why this isn't failing anymore. I guess jobs.query is more lenient about unknown parameters than jobs.insert is?

Reply:
Could you share a link to a failing test log? I wasn't able to find it in test fusion.

Reply:
Should we keep this test?

Reply:
I tested locally with https://github.com/googleapis/python-bigquery/pull/1793 and the request body now includes unknown parameters, but surprisingly the REST API seems to ignore these. I'll follow-up on that PR to raise on common errors, such as passing in a configuration object of the wrong type.

Reply:
Updated https://github.com/googleapis/python-bigquery/pull/1793 with some client-side checks for common mistakes of sending the wrong type for `job_config` and now this test is passing.

Will need to wait for a `google-cloud-bigquery` release with that fix before merging this PR.